src/modules/dashboard/components/guides/Regular.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="regular-guideline">
+    <span>Create your token (or its available variations: NFT and SBT)</span>
+    <p>Launch your unique token efficiently and securely using the established infrastructure of the SORA Network.</p>
+  </div>
+</template>
+
+<script lang="ts">
+import { mixins } from '@soramitsu/soraneo-wallet-web';
+import { Mixins } from 'vue-property-decorator';
+
+export default class RegularGuide extends Mixins(mixins.TranslationMixin) {}
+</script>
+
+<style lang="scss">
+.regular-guideline {
+  min-height: 372px;
```

Comment:
Why this min-height? From where we got this number?

Reply:
if you switch tabs you won't notice any height differences between SBT/NFT. (No UI elements fluctuations)